safe/report/test/test_impact_report.py

```diff
@@ -1063,6 +1065,152 @@ def test_minimum_needs_outputs(self):
 
         shutil.rmtree(output_folder, ignore_errors=True)
 
+    def test_minimum_needs_outputs_modified(self):
+        """Test generate minimum needs section with updated profile
+
+        .. versionadded: 4.3
+        """
+        # Make sure that the displacement rate is the correct one that has
+        # been set in setUp method.
+        displacement_rate = get_displacement_rate(
+            hazard_flood['key'], flood_hazard_classes['key'], 'wet')
+        self.assertEqual(displacement_rate, self.custom_displacement_rate)
+
+        # Change affected
+        profile = setting(key='profile')
+        profile[hazard_flood['key']][flood_hazard_classes['key']][
+            'wet']['affected'] = False
+        profile[hazard_flood['key']][flood_hazard_classes['key']][
+            'dry']['affected'] = True
+        profile[hazard_flood['key']][flood_hazard_classes['key']][
+            'dry']['displacement_rate'] = 0.5
+        set_setting(key='profile', value=profile)
+        is_affected_wet = is_affected(
```

Comment:
I took me more seconds to read, `wet_is_affected` looks better. Same for the one. But I also don't care, it's a test file. For next time